h/static/scripts/annotator/anchoring/types.coffee

```diff
@@ -0,0 +1,55 @@
+Annotator = require('annotator')
+$ = Annotator.$
+xpathRange = Annotator.Range
+
+
+# Helper function for throwing common errors
+missingParameter = (name) ->
+  throw new Error('missing required parameter "' + name + '"')
+
+
+###*
+# class:: RangeAnchor(range)
+#
+# This anchor type represents a DOM Range.
+#
+# :param Range range: A range describing the anchor.
+###
+class RangeAnchor
```

Comment:
Let's leave it in for now but I'm totally up for removing it later. Certainly if it's not useful above and beyond the position selectors, then given that it's the cause of some of Annotator's more idiosyncratic (read: annoying) behaviour, such as splitting textNodes in the DOM when you make selections, I'd be all for removing it once we've checked the database to ensure it's not needed.